plugins/languages/treesitter/treesitter.nix

```
@@ -142,12 +142,21 @@ helpers.neovim-plugin.mkNeovimPlugin config {
       List of parsers to ignore installing. Used when `ensure_installed` is set to "all".
     '';
 
-    parser_install_dir = helpers.defaultNullOpts.mkStr "$XDG_DATA_HOME/nvim/treesitter" ''
-      Location of the parsers to be installed by the plugin (only needed when `nixGrammars` is disabled).
+    parser_install_dir = helpers.mkNullableWithRaw' {
+      type = types.str;
```

```suggestion
    parser_install_dir = helpers.mkNullOrOption' {
      type = with helpers.nixvimTypes; maybeRaw str;
```

My mistake (as discussed on matrix)